sources/osg/Matrix.js

```diff
@@ -1170,8 +1341,94 @@ define( [
             };
         } )(),
 
-        // compute the 4 corners vector of the frustrum
-        computeFrustrumCornersVectors: function ( projectionMatrix, vectorsArray ) {
+        clampProjectionMatrix: function ( projection, znear, zfar, nearFarRatio, resultNearFar ) {
```

Comment:
Did you move the code from CullVisitor here because you use it in another context than CullVisitor ? Could you show me where ?